app/src/main/java/app/abhijit/iter/LoginActivity.java

```diff
@@ -37,6 +37,7 @@
 import android.support.v7.widget.Toolbar;
 import android.text.Editable;
 import android.text.TextWatcher;
+import android.util.Log;
```

Comment:
Can you please remove this import?

Reply:
sure
Can I change something in this PR? Or do I need to make another one?

Reply:
Just make another commit and push it.

Reply:
> Just make another commit and push it.

okay